public/src/components/admin/MarketState.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { Metadata } from "grpc-web-client";
+import { showNotif, showErrorNotif, closeNotifs } from "../../utils";
+import { DalalActionService } from "../../../proto_build/DalalMessage_pb_service";
+import { OpenMarketRequest } from "../../../proto_build/actions/OpenMarket_pb";
+import { CloseMarketRequest } from "../../../proto_build/actions/CloseMarket_pb";
+
+export interface MarketStateProps{
+    sessionMd: Metadata,
+    isMarketOpen: boolean
+}
+export interface MarketStateState{
+    option: string
```

Comment:
Change this from a generic variable to something that conveys its use like `openMarket`. Also make the variable a `boolean `rather than `string`.